docs/source/technical_design/debianinstall.rst

```diff
@@ -254,7 +254,7 @@ To start KAT when the system boots, enable all KAT services:
 Start using OpenKAT
 ===================
 
-By default OpenKAT will be accessible in your browser through `https://<server IP>:8000`. There, Rocky will take you through the steps of setting up your account and running your first boefjes.
+By default OpenKAT will be accessible in your browser through `https://<server IP>:8443` (:8000 for docker based installs). There, Rocky will take you through the steps of setting up your account and running your first boefjes.
```

Comment:
voor docker is het http, ipv https.

`https://<server IP>:8443` for debian packages or `http://<server IP>:8000 for docker based installs.